mono/utils/mono-error.c

```diff
@@ -360,16 +354,13 @@ mono_error_set_type_load_name (MonoError *oerror, const char *type_name, const c
 }
 
 void
-mono_error_set_method_load (MonoError *oerror, MonoClass *klass, const char *method_name, const char *signature, const char *msg_format, ...)
+mono_error_set_method_load (MonoError *oerror, const char *message)
```

Comment:
should we rename that to `mono_error_set_missing_method` so it's closer to the exception we are going to throw?